### PR TITLE
Add python 3.5 compatibility

### PR DESCRIPTION
**Pull request by staticshock**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -5,14 +5,16 @@ max-line-length = 100
 
 [tox]
 install_command = pip install {opts} {packages}
-envlist = lint,clean,py27-{1.7,1.8,1.9}-{postgres,sqlite},stats
+envlist = lint,clean,py27-{1.7,1.8,1.9}-{postgres,sqlite},py35-{1.8,1.9}-{postgres,sqlite},stats
```
Comment (lucaswiman) [inline tox.ini]: Out of curiosity, are you using Django 1.7? I'd like to drop support for that once my work (finally) upgrades, hopefully in the next month or so.

File: tox.ini [hunk 2]
```diff
@@ -5,14 +5,16 @@ max-line-length = 100
...
 
 [testenv]
 usedevelop = True
 commands =
   {envbindir}/coverage run -p --omit="*tests*" --source=predicate --branch \
     setup.py test
-basepython = python2.7
```
Comment (lucaswiman) [inline tox.ini]: This seems to have broken the `lint` tox environment. I think you need to define a `basepython` parameter. How about updating this to:
```
basepython =
    python3.5
    py27: python2.7
```

**Comment by lucaswiman** [comment]
This is great! Should be good to merge after the `basepython` thing is fixed.

**Comment by lucaswiman** [comment]
Merging this in so I can cut a release once #33 merges.

**Comment by lucaswiman** [comment]
Deployed to pypi: https://pypi.python.org/pypi/django-predicate/1.4.0

File: tox.ini (hunk 2 above)
Comment (staticshock) [inline tox.ini]: The lint tox environment was broken for me before this change, actually. I didn't try to fix it.

File: tox.ini (hunk 1 above)
Comment (staticshock) [inline tox.ini]: I was using Django 1.7 until a couple of months ago.

**Pull request merged**